manifests/repository.pp

```diff
@@ -4,7 +4,8 @@
   $yum_proxy_password   = undef,
   $yum_deps_baseurl     = undef,
   $yum_products_baseurl = undef,
-  ){
+  $apt_repo             = 'puppet',
```

Comment:
why puppet and not main?

Reply:
Because there is no "main" repository for most distros on apt.puppetlabs.com:

https://apt.puppet.com/dists/stretch/
https://apt.puppet.com/dists/bionic/

It also matches the ``sources.list.d`` files installed by the various puppet-*-release.deb packages.

Reply:
ack thanks